Add babel-inline-import-loader in webpack

gchoqueux opened the pull request:
## Motivation
A webpack loader enabling files imported by babel-plugin-inline-import to trigger rebuilds when content changes.

gchoqueux committed:
chore: clean up webpack module.exports

tbroyer reviewed (commented):
6 commits, 1 published version, last published 3 years ago :thinking: Not the kind of dependency I'd add in my projects…

BTW, does it play well with `babel-plugin-inline-import` that we already have in `.babelrc`? That one points to https://github.com/elliottsj/babel-inline-import-loader when using webpack.

gchoqueux commented:
I had seen that it was an old fossil, I hesitated ;)
Thank you, your proposal seems more logical.

gchoqueux commented:
@tbroyer
I replaced by `babel-inline-import-loader`

gchoqueux committed:
chore: add babel-inline-import-loader in webpack

A webpack loader enabling files imported by babel-plugin-inline-import to trigger rebuilds when content changes.

zarov commented on webpack.config.js:
```diff
@@ -29,10 +29,17 @@ babelConf.babelrc = false; // disabel babelrc reading, as we've just done it
+];
+
 module.exports = {
+    context: path.resolve(__dirname),
```
Weird from webpack that we can't use this context everywhere :thinking:

gchoqueux replied:
some options need absolute path...

zarov reviewed (approved):
This is welcome !
